api/managment/views.py

```diff
@@ -97,14 +89,30 @@ class TranzactionViewSet(viewsets.ModelViewSet):
 
 class DoctorViewSet(viewsets.ModelViewSet):
     serializer_class = DoctorSerializer
-    permission_classes = [IsAuthenticated]
+    # permission_classes = [IsAuthenticated]
+
+    def get_queryset(self):
+        type_user = get_type_of_user(self.request.user)
+        if type_user == "guardian":
+            try:
+                ward = int(self.request.GET.get('ward'))
+            except Exception:
+                raise Exception("404 bad ward")
+            return Doctor.objects.filter(patient__user=ward)
```

Comment:
Rewrite logic of endpoint to be able to use
`Doctor.objects.filter(patient=ward)`